crates/hyperdrive-math/src/short/open.rs

```diff
@@ -754,20 +758,27 @@ mod tests {
 
             // Bob closes his short.
             let base_proceeds = bob.close_short(maturity_time, bond_amount, None).await?;
+            let position_duration = bob.get_state().await?.annualized_position_duration();
```

Comment:
I would call this `annualized_position_duration` to avoid confusion from the `position_duration` attribute.

Reply:
Updated, thanks!